spring-boot/sap-idoc-destination-spring-boot/pom.xml

```diff
@@ -109,8 +138,73 @@
 					</execution>
 				</executions>
 			</plugin>
-
 		</plugins>
-
 	</build>
+	<profiles>
+		<profile>
+			<id>openshift</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.eclipse.jkube</groupId>
+						<artifactId>openshift-maven-plugin</artifactId>
+						<configuration>
+							<resources>
+								<controller>
+									<env>
+										<AB_PROMETHEUS_OFF>true</AB_PROMETHEUS_OFF>
+										<AB_JOLOKIA_OFF>true</AB_JOLOKIA_OFF>
+										<AB_ENABLED>false</AB_ENABLED>
+										<JAVA_MAIN_CLASS>org.springframework.boot.loader.PropertiesLauncher</JAVA_MAIN_CLASS>
+										<JAVA_OPTIONS>-Dloader.main=${main.class}</JAVA_OPTIONS>
+									</env>
+								</controller>
+							</resources>
+							<images>
+								<image>
+									<name>camel/actuator-http-metric</name>
+									<build>
+										<from>quay.io/jkube/jkube-java:0.0.19</from>
```

Comment:
can this image parameterized? it would be useful in case we want to test different images, thanks